zeppelin/interpreters/flink.rst

```diff
@@ -0,0 +1,37 @@
+==================
+Flink Interpreter
+==================
+
+Flink interpreter for Apache Zeppelin
+--------------------------------------
+
+`Apache Flink <https://flink.apache.org/>`_ is an open source platform for distributed stream and batch data processing. Flink’s core is a streaming dataflow engine that provides data distribution, communication, and fault tolerance for distributed computations over data streams. Flink also builds batch processing on top of the streaming engine, overlaying native iteration support, managed memory, and program optimization.
+
```

Comment:
Instead of describing what Flink is, you can point to the documentation page and be more specific about the role of the interpreter and how it allows to comfortably use Flink in the browser to explore data sets.